### PR TITLE
feat(board-stats)[OK-339] Add getTimesSpentOnBoardInMs endpoint

### PR DESCRIPTION
**Pull request by Wajktor13**

### CODE REVIEW

File: server/src/modules/users/users.module.ts [hunk 1]
```diff
@@ -15,6 +15,6 @@ import { User, UserSchema } from '../../mongo/schemas/user.schema';
   ],
   controllers: [UsersController],
   providers: [UsersService],
-  exports: [UsersService],
+  exports: [UsersService, MongooseModule],
```
Comment (marcepanowyy) [inline server/src/modules/users/users.module.ts]: Why is it exporting mongoose module?

**Review by marcepanowyy — CHANGES_REQUESTED** [review]

File: server/src/modules/users/users.module.ts (hunk 1 above)
Comment (Wajktor13) [reply server/src/modules/users/users.module.ts]: I get this without:

image

File: server/src/modules/stats/board/board.stats.module.ts [hunk 2]
```diff
@@ -15,9 +17,10 @@ import {
         schema: BoardStatsSchema,
       },
     ]),
+    UsersModule,
   ],
   controllers: [BoardStatsController],
-  providers: [BoardStatsService],
+  providers: [BoardStatsService, UsersService],
```
Comment (marcepanowyy) [inline server/src/modules/stats/board/board.stats.module.ts]: UsersService should not be a provider of this module. Currently it is a provider of UsersModule and it cannot serve as a second provider

File: server/src/modules/users/users.module.ts (hunk 1 above)
Comment (marcepanowyy) [reply server/src/modules/users/users.module.ts]: no module should export MongooseModule - it should be only imported

**Review by marcepanowyy — CHANGES_REQUESTED** [review]
1) Delete `UsersService` from providers of `BoardStatsModule`
2) Delete `MongooseModule` from exports of `UsersModule`

It will work, however another fix should be made to get rid of `MongooseModule` from `exports` in `BoardStatsModule`

If you have problem with dependencies just draw them - it may simplify debugging

**Review by marcepanowyy — APPROVED** [review]

**Pull request merged**